blog/2024-09-04-lawn-mower-activity-returning.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Michael Arthur
+authorURL: https://github.com/mikey0000
+title: "New returning type in LawnMowerActivity"
+---
+
+As of Home Assistant Core 2024.9, integrations using the lawn_mower platform can indicate that their mower 
+is returning to the dock by using the new `LawnMowerActivity.RETURNING` state.
```

Comment:
Please add a link to the dev docs page of the lawnmower entity.

https://developers.home-assistant.io/docs/core/entity/lawn-mower#activities

Reply:
done